android/app/src/main/java/org/fossasia/openevent/activities/TrackSessionsActivity.java

```diff
@@ -134,6 +143,33 @@ private void loadData() {
 
             mSessions.clear();
             mSessions.addAll(track.getSessions().sort("startsAt"));
+
+//            finding upcoming and ongoing sessions
```

Comment:
Do you mean that they aren't required here or that I should place them somewhere else. Could you be a bit more specific please?